fpsim/locations/ethiopia.py

```diff
@@ -409,6 +463,19 @@ def sexual_activity():
 
     return activity_interp
 
+def sexual_activity_region():
+    '''
+    Returns a linear interpolation of rates of female sexual activity, stratified by region
+    '''
+
+    sexual_activity_region = pd.read_csv(thisdir / 'ethiopia' / 'subnational' / 'urban.csv')
+    sexual_activity_region['region'] = sexual_activity_region[:, 0] # Return region names
```

Comment:
similar feedback here - I would name the data read in from the csv as a distinct variable from the sexual activity data you're storing (i.e. sexual_activity_region['region'])

Reply:
Same as above - variable name has been updated.